mlrun/utils.py

```diff
@@ -430,3 +431,43 @@ def fill_function_hash(function_dict, tag=''):
     function_dict['metadata']['hash'] = hashkey
     function_dict['status'] = status
     return hashkey
+
+
+def retry_until_successful(interval: int, timeout: int, logger, verbose: bool, function, *args, **kwargs):
```

Comment:
default verbose to False

Reply:
it'll be problematic because of the after the function **kwargs parameter